homeassistant/helpers/entity_component.py

```diff
@@ -37,6 +37,7 @@
 async def async_update_entity(hass: HomeAssistant, entity_id: str) -> None:
     """Trigger an update for an entity."""
     domain = entity_id.split(".", 1)[0]
+    entity_comp: EntityComponent[entity.Entity] | None
```

Comment:
I had an issue with a test I was debugging, and adding the type hint here helped me figure out the issue in the test.